docs/developer-docs/smart-contracts/maintain/state.mdx

```diff
@@ -7,39 +7,18 @@ import { GlossaryTooltip } from "/src/components/Tooltip/GlossaryTooltip";
 
 # State
 
-<MarkdownChipRow labels={["Beginner", "Tutorial"]} />
+<MarkdownChipRow labels={["Beginner", "Concept"]} />
 
 ## Overview
 
 After you deploy a <GlossaryTooltip>canister</GlossaryTooltip>, it can begin receiving and processing requests from users and from other canisters. Canisters that are available to send requests and receive replies are considered in be in a **Running** state.
 
 Although canisters are normally placed in the Running state by default, there are cases where you might want to temporarily or permanently stop a canister. For example, you might want to stop a canister before upgrading it. Stopping a canister helps to ensure proper handling of any messages that are in progress and need to either run to completion or be rolled back. You might also want to stop a canister to clear its message queue cleanly as a prerequisite to deleting the canister.
 
-You can check the current status of all canisters or a specified canister by running the `dfx canister status` command. For example, to see the status for all canisters running on the local canister execution environment, you would run the following command:
+To modify the state of a canister:
 
-    dfx canister status --all
+- [`dfx canister status <canister-name>`](/docs/current/developer-docs/developer-tools/cli-tools/cli-reference/dfx-canister#dfx-canister-status): Check if the canister is Running, Stopping, or Stopped. This command also returns details about the canister's settings.
 
-This command returns output similar to the following if canisters are currently running:
+- [`dfx canister stop <canister-name>`](/docs/current/developer-docs/developer-tools/cli-tools/cli-reference/dfx-canister#dfx-canister-stop): Stop the canister. The canister's status will change to Stopping if it has in-flight messages that must be addressed before it can be stopped.
```

Comment:
```suggestion
- [`dfx canister stop <canister-name>`](/docs/current/developer-docs/developer-tools/cli-tools/cli-reference/dfx-canister#dfx-canister-stop): Stop the canister. The canister's status will change to `Stopping` if it has in-flight messages that must be addressed before it can be stopped.
```